cclib/parser/orcaparser.py

```diff
@@ -1339,9 +1339,61 @@ def energy_intensity(line):
                     etoscs.append(float(intensity))
 
                     line = next(inputfile)
-
-                self.set_attribute('etenergies', etenergies)
-                self.set_attribute('etoscs', etoscs)
+                
+                # Some of these sections contain data that we probably do not want to be populating etenergies
+                # and/or etoscs with.  For example, the SOC corrected spectra are for mixed singlet/triplet states,
```

Comment:
Aye that makes sense. Something that I'm increasingly thinking would be a useful change is to split multiplicity and symmetry into separate attributes (as you say they are distinct properties). Technically this might not be backwards compatible though (if we're taking multiplicity OUT of the symmetry attribute), so might need to wait...